src/test/java/de/zalando/zmon/dataservice/proxies/entities/EntitiesControllerTest.java

```diff
@@ -74,7 +74,7 @@ public void getEntities() throws Exception {
 
     @Test
     public void getEntitiesRest() throws Exception {
-        this.mockMvc.perform(MockMvcRequestBuilders.get("/rest/api/v1/entities?query=htg").header("Authorization", "Bearer 1"))
```

Comment:
Shouldn't we keep legacy path test to ensure compatibility? (until it is removed)